### PR TITLE
Photos

### PR DESCRIPTION
**Pull request by prestonp**

### CODE REVIEW

**Comment by prestonp** [comment]
photo-carousel

**Comment by prestonp** [comment]
You'll probably notice that there's a lot of photos reused in multiple parks. Not only that, lots of that photos are completely unrelated. 

I'm still tweaking the results from https://github.com/open-austin/austin-parks-photos, but this seems like a decent start

**Comment by prestonp** [comment]
I think replacing the carousel with something http://packery.metafizzy.co/ + https://github.com/fat/zoom.js would be really slick

File: client/js/components/ParkPhotos.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        this.setState({idx: idx});
+    }
+
+    prev() {
+        const idx = this.state.idx - 1 < 0 ? this.state.photos.length-1 : --this.state.idx;
+        this.setState({idx: idx});
+    }
+
+    render() {
+        if (!this.state.photos.length) return (<div></div>);
```
Comment (luqmaan) [inline client/js/components/ParkPhotos.jsx]: Need those curly braces.

``` js
if (!this.state.photos.length) { return (<div></div>); }
```

**Comment by luqmaan** [comment]
:deciduous_tree:

That was fast!

**Comment by luqmaan** [comment]
I like the zoom.js idea.

What do you think of combining it with a list of the thumbnails like this:

image

Clicking on a thumbnail expands it with zoom.js.

I like the _q thumbnail size: https://www.flickr.com/services/api/misc.urls.html

File: client/js/components/ParkPhotos.jsx (hunk 1 above)
Comment (prestonp) [inline client/js/components/ParkPhotos.jsx]: Gotcha

**Comment by prestonp** [comment]
I like the thumbnails + zoom.js plugin, I'll add those soon

**Comment by luqmaan** [comment]
:deciduous_tree: merge whenever you're ready

**Comment by prestonp** [comment]
screen shot 2015-07-30 at 8 34 33 pm

**Comment by luqmaan** [comment]
can you do this without a whole new font and theme? All you need is a chevron svg

**Comment by luqmaan** [comment]
The chevron component does this for example. Just change the rotation transform.

**Comment by prestonp** [comment]
Yeah I just pulled in from the lib, I can trim that stuff down

**Comment by luqmaan** [comment]
I think react-slick is overkill for what we need.

To create the left/right sliding gallery we could use a div with `overflow: scroll`. When the user clicks on the arrow, scroll the div to the left or the right. Or instead use `position: absolute` and animate the left position of the images. 

To create the arrows use the svg from here https://github.com/open-austin/austingreenmap/blob/ef5f5d3b6cef32933d955b3658cb25d14eb4de5a/client/js/components/Chevron.jsx#L3. Then apply a css transform to rotate the arrow.

Sorry for being nitpicky, but this slick stuff is so heavy for what we need:

```
 client/fonts/slick.eot              | Bin 0 -> 2048 bytes
 client/fonts/slick.svg              |  14 ++++++++++
 client/fonts/slick.ttf              | Bin 0 -> 1892 bytes
 client/fonts/slick.woff             | Bin 0 -> 1380 bytes
 client/scss/slick-theme.scss        | 187 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 client/scss/slick.scss              |  99 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 client/scss/theme.scss              |  13 ++++++++++
```

Does this make sense?

Should we just roll with react-slick and revisit the photo section later?

**Comment by prestonp** [comment]
Hey it's no prob at all, I figured my attempts to build a carousel wouldn't
be as slick (pun intended)

I'll try building a lighter version, in the meantime I'll close this pr

**Comment by luqmaan** [comment]
Hah, nice pun pham.

**Pull request closed without merging**